news/_posts/releases/2019-05-13-prestashop-1-7-6-0-beta-release.md

```diff
@@ -42,50 +42,54 @@ If you find regressions compared to previous versions, please create an issue on
 
 ## News & improvements
 
-### New features & improvements
+### SEO Optimization
 
--   SEO optimization for combination products: all combinations now have the parent product’s URL as their canonical, accessing it no longer issues a redirection, displays the default combination [\#13009](https://github.com/PrestaShop/PrestaShop/pull/13009)
--   A “Price tax included” column has been added to the products list [\#11484](https://github.com/PrestaShop/PrestaShop/pull/11484)
--   A new “modern” template design & wording for transactional emails and a
-    new email template generation system[\#13004](https://github.com/PrestaShop/PrestaShop/pull/13004)
+SEO optimization for combination products:
+- All combinations now have the parent product’s URL as canonical,
+- Going the the parent product’s URL displays the default combination, there no redirection anymore
```

Comment:
Going to the parent product’s URL displays the default combination, there is no redirection anymore